.github/workflows/tests.yaml

```diff
@@ -194,3 +194,77 @@ jobs:
             - name: Upload coverage
               uses: codecov/codecov-action@v1
               if: matrix.python-version == '3.13'
+              
+    mysql:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                python-version: ["3.10", "3.11", "3.12", "3.13", "3.14"]
+
+        services:
+            mysql:
+                image: mysql:8.4
+                env:
+                    MYSQL_ROOT_PASSWORD: rootpassword
+                    MYSQL_DATABASE: piccolo
+                    MYSQL_ROOT_HOST: '%'
+                options: >-
+                    --health-cmd="mysqladmin ping -uroot -prootpassword"
+                    --health-interval=5s
+                    --health-timeout=2s
+                    --health-retries=10
+                ports:
+                    - 3306:3306
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v4
+
+            - name: Set up Python ${{ matrix.python-version }}
+              uses: actions/setup-python@v5
+              with:
+                python-version: ${{ matrix.python-version }}
+
+            - name: Install dependencies
+              run: |
+                python -m pip install --upgrade pip
+                pip install -r requirements/requirements.txt
+                pip install -r requirements/test-requirements.txt
+                pip install -r requirements/extras/mysql.txt
+
+            - name: Install MySQL Client
+              run: sudo apt-get update && sudo apt-get install -y mysql-client
+
+            - name: Wait for MySQL
+              run: |
+                set -e
+                for i in {1..60}; do
+                    if mysqladmin ping -h127.0.0.1 -P3306 -uroot -prootpassword > /dev/null 2>&1; then
```

Comment:
Can there be spaces here - so `-h 127.0.0.1` instead of `-h127.0.0.1`?

Reply:
That doesn't work. I'll revert to the previous commands.